src/applications/mhv-medical-records/containers/ConditionDetails.jsx

```diff
@@ -68,12 +70,30 @@ const ConditionDetails = props => {
     [dispatch],
   );
 
+  const { isAcceleratingConditions } = useAcceleratedData();
+
   useEffect(
     () => {
-      if (conditionId)
-        dispatch(getConditionDetails(conditionId, conditionList));
+      if (conditionId && !record?.notFound)
+        dispatch(
+          getConditionDetails(
+            conditionId,
+            conditionList,
+            isAcceleratingConditions,
+          ),
+        );
+      if (record?.notFound) {
+        history.push('/conditions');
+      }
```

Comment:
Single Conditions endpoint is not available - If an detail item is not found, the user is redirected to the Conditions list page

Reply:
The non-accelerated experience remains the same:
* Fetch the item from the list; if not found,
* Fetch the single item from the endpoint (by ID)

Since a fetch-by-ID endpoint isn’t available for the accelerated experience, we’re following the same approach as in the accelerated Labs and Tests domain. Do you think we should explore alternatives?